src/Debugger/Impl/rtvs/R/grid.R

```diff
@@ -72,3 +72,79 @@ grid.dput <- function(obj) {
     json;
 }
 
+getEnvRepr <- function(env, startEnvLevel) {
+  envRepr <- NULL;
+
+  if (startEnvLevel > 4L) {    # if within call or unknown, try to get call frame
+    nframe <- sys.nframe();
+    if (nframe > 0) {
+      for (i in 1:nframe) {
+        frame <- sys.frame(i)
+        if (identical(env, frame)) {
+          envRepr <- list(name = deparse(sys.call(i)[[1]]), level = 5L, frameIndex = i);
+          break;
+        }
+      }
+    }
+  }
+
+  if (is.null(envRepr)) {
+    if (identical(env, baseenv())) {
```

Comment:
But it will return the name for `globalenv()`, `baseenv()` etc, which is what is being checked for here.

Better yet, use `search()`, iterate over names it returns, and use `as.environment` to get the corresponding environment objects (this will produce "package:base" rather than "base" for the base environment, among other things).